include/zephyr/arch/cache.h

```diff
@@ -89,6 +89,9 @@ extern int arch_dcache_flush_and_invd_all(void);
  *
  * Flush the specified address range of the data cache.
  *
+ * @note: when @p addr is not aligned to the cache line and/or @p size is not
```

Comment:
```suggestion
 * @note when @p addr is not aligned to the cache line and/or @p size is not
```

forgot this one (applies to all others)